src/components/app.jsx

```diff
@@ -44,6 +44,9 @@ const App = React.createClass({
       searchResults: findParametersAndVariables(parameters, variables, searchQuery),
     }
   },
+  isCurrentRoute(route) {
+    return this.props.location.pathname == route
```

Comment:
This works in this case because routes have no param, but I think the idiomatic way to match a path is with [`matchPath`](https://reacttraining.com/react-router/web/api/matchPath) :

```js
matchPath(route, this.props.location)
```

Since I'm not sure could you just try?